Week1/homework/js-exercises/roundNumber.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+let z = 7.25;
+console.log("z =  "+ z);
+let a = Math.round(z);
+console.log("a =  "+a);
+let hight;
+    if(z > a){
+    hight = z
+    console.log("to z einai megalitero " +hight);
+
+    }else {
+        hight = a;
+        console.log("to a einai megalitero " +hight);
+    }
```

Comment:
Works great!
Just a side note regarding the styling...
The indentation should only apply right after the `if statement`.  This makes it easy to identify what is part of the `if statement`.

i.e.
```
let hight;
if (z > a) {
 hight = z
 console.log("to z einai megalitero: " + hight);
} else {
 hight = a;
 console.log("to a einai megalitero: " + hight);
}
```